pkg/fixtures/fixtureconsts/fixture_consts.go

```diff
@@ -46,4 +46,7 @@ const (
 	NetworkPolicy2      = "ffaaaaaa-0000-0000-0000-111111111111"
 	NetworkPolicy3      = "ffaaaaaa-0000-0000-0000-222222222222"
 	NetworkPolicy4      = "ffaaaaaa-0000-0000-0000-333333333333"
+
+	PodName1 = "nginx-7db9fccd9b-92hfs"
```

Comment:
nit:  line this up for readability.

Reply:
I made the change, but the style check failed. Running `make style` locally reverts the change. If I take out the blank line the alignment is allowed. I did want to have the blank line to distinguish the names from the UUIDs.

Reply:
That makes sense.  It is good as is, then.